packages/ffe-core-react/src/wave/wave.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { string, bool } from 'prop-types';
+import classNames from 'classnames';
+
+export default function Wave(props) {
+    const { size, rotate, color, ...rest } = props;
+
+    return (
+        <div
+            aria-hidden="true"
+            className={classNames('ffe-wave', `ffe-wave--${size}`, {
+                'ffe-wave--rotate': rotate,
+            })}
+            style={{ backgroundColor: `var(--ffe-farge-${color})` }}
+            {...rest}
+        />
+    );
+}
+
+Wave.propTypes = {
+    size: string,
+    rotate: bool,
+    color: string,
```

Comment:
da må alla ha isDarkMode is javascript. Vi har det men har alla andra det?

```
color: {
   starndard: string,
   dark: string 
}
```

Reply:
Må se litt nærmere på hvordan vi kan løse dette, fordi vi har ingen andre komponenter (så vidt jeg vet) som tar inn både darkmode og lightmode farger.  I messageBox f.eks så mapper vi opp at x farge er Y i darkmode f.eks.  Men litt usikker på om det blir riktig å gjøre her.

Hvis vi har en dark og en light farge property, så må vi finne ut av hvem man skal bruke i selve komponenten,  og jeg er litt usikker på om alle gjør det på samme måte. For om folk gjør det på samme måte går det bra, men hvis folk sjekker på litt forskjellige måter er kanskje det beste å bare la folk "sjekke selv".